### DISCUSSION TITLE
Serverless Functions Issue

### DISCUSSION THREAD

**Discussion by dpaulos6** [vercel/community#4990] [Help] [answered]
### Summary

I'm currently using Astro and i'm trying to setup nodejs serverless functions, and I can't make my setup work.

Folder structure
```txt
+---.vercel
+---dist
+---node_modules
+---public
+---src
    +---api
    +---assets
    ¦   +---images
    ¦   ¦   +---gallery
    ¦   +---logos
    +---components
    +---layouts
    +---pages
    +---scripts
    +---styles
...
```

/api/text.js
```js
export default function handler(request, response) {
  const { name = 'World' } = request.query;
  return response.send(`Hello ${name}!`);
}
```

when I visit `http://localhost:3000/api/test?query=Username` I get a 404

### Steps to Reproduce

- Run `vercel dev`
- Acess `http://localhost:3000/api/test`
- Displays `404: Not found`

**Comment by thomasledoux1** [comment, marked as answer]
Have you tried moving the /api folder to the root level instead of in the /src folder?
